README.md

```diff
@@ -0,0 +1,19 @@
+## YaST - Sudo Configuration
+
+Intention of this module is to provide UI for configuring sudo.
+
+### Known Limitations
+
+- Module using for parsing sudoers own handcrafted perl parser which has
+  limitations especially regarding new sudo changes as opposite to e.g.
+  augeas lenses that are more up-to data.
+- Support for `@include` and `@includedir` directive is limited. It just
+  shows them in rules section, but nothing more. Also duplicite alias detection
+  does not work across include files.
+- Sudo configuration does not support multitag in rule, which leads to dropping
+  it ( caused by limitation of parser, see above )
```

Comment:
Sounds like a bug. Can we prevent from that?

Reply:
OK, so parser is not limitation here as it return whole part as string https://github.com/yast/yast-sudo/blob/master/src/servers_non_y2/ag_etc_sudoers#L30
problem is in fact in separt part of parsing done in module which read just one TAG:
https://github.com/yast/yast-sudo/blob/master/src/modules/Sudo.rb#L175
and of course list of tags is limited, sudo supports much more tags.

Reply:
I try to fix it, but failed misserably, as I found another limitation with command specific tags and for it, we do not have prepared data structures ( we have tags list and command list, but not that tag is associated with command ).